Libraries/Modal/Modal.js

```diff
@@ -153,16 +154,19 @@ class Modal extends React.Component {
   static contextTypes = {
     rootTag: PropTypes.number,
   };
+
   constructor(props: Object) {
     super(props);
     Modal._confirmProps(props);
   }
+
   componentWillReceiveProps(nextProps: Object) {
     Modal._confirmProps(nextProps);
   }
+
   static _confirmProps(props: Object) {
     if (props.presentationStyle && props.presentationStyle !== 'overFullScreen' && props.transparent) {
-      console.warn(`Modal with '${props.presentationStyle}' presentation style and 'transparent' value is not supported.`);
+      warning(false, `Modal with '${props.presentationStyle}' presentation style and 'transparent' value is not supported.`);
```

Comment:
Yeah, but it should be something like

`warning(!(props.presentationStyle && props.presentationStyle !== 'overFullScreen' && props.transparent), ...)`
or
`warning(!props.presentationStyle || props.presentationStyle === 'overFullScreen' || !props.transparent, ...)`
.